<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>781</width>
     <height>571</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="resourcesTab">
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <attribute name="title">
     <string>&amp;Resources</string>
    </attribute>
    <widget class="QWidget" name="cpu_widget" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>740</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="cpu_usage">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>350</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>cpu_usage</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>400</y>
       <width>741</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Memory Usage</string>
     </property>
     <widget class="QLabel" name="memoryInfo">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>40</y>
        <width>471</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Mem_info</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>82</y>
        <width>471</width>
        <height>31</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Memory used:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Memory Infomation:</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="processesTab">
    <attribute name="title">
     <string>&amp;Processes</string>
    </attribute>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>160</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Sleep:</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView_pro">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>551</width>
       <height>381</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>280</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Zombie:</string>
     </property>
    </widget>
    <widget class="QLabel" name="pro_zom">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>280</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>000</string>
     </property>
    </widget>
    <widget class="QLabel" name="pro_io">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>220</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>000</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>40</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Total:</string>
     </property>
    </widget>
    <widget class="QLabel" name="pro_sleep">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>160</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>000</string>
     </property>
    </widget>
    <widget class="QPushButton" name="proRefBtn">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>350</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Re&amp;fresh</string>
     </property>
    </widget>
    <widget class="QLabel" name="pro_total">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>40</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>000</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>100</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Run:</string>
     </property>
    </widget>
    <widget class="QLabel" name="pro_run">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>100</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>000</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>220</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>I/O:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="findBtn">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>430</y>
       <width>89</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Fin&amp;d</string>
     </property>
    </widget>
    <widget class="QPushButton" name="killBtn">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>480</y>
       <width>89</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&amp;Kill</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>410</y>
       <width>641</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLineEdit" name="lineEditPid">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>551</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditName">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>551</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>67</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>67</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>PID:</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="systemTab">
    <attribute name="title">
     <string>&amp;System</string>
    </attribute>
    <widget class="QGroupBox" name="osInfo">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>721</width>
       <height>247</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>OS info</string>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Hostname:</string>
      </property>
     </widget>
     <widget class="QLabel" name="hostname">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>60</y>
        <width>511</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>hostname</string>
      </property>
     </widget>
     <widget class="QLabel" name="ostype">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>100</y>
        <width>511</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>ostype</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Type:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>140</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Version:</string>
      </property>
     </widget>
     <widget class="QLabel" name="osversion">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>140</y>
        <width>511</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>osversion</string>
      </property>
     </widget>
     <widget class="QLabel" name="gccversion">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>180</y>
        <width>511</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>gccversion</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>180</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>GCC version:</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="cpuInfo">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>280</y>
       <width>721</width>
       <height>246</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>CPU info(each core)</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Type:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>140</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Frequency:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>180</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Cache size:</string>
      </property>
     </widget>
     <widget class="QLabel" name="cputype">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>100</y>
        <width>511</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>cputype</string>
      </property>
     </widget>
     <widget class="QLabel" name="cpuname">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>60</y>
        <width>511</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>cpuame</string>
      </property>
     </widget>
     <widget class="QLabel" name="cpucache">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>180</y>
        <width>511</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>cpucache</string>
      </property>
     </widget>
     <widget class="QLabel" name="cpufre">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>140</y>
        <width>511</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>cpufrequency</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="aboutTab">
    <attribute name="title">
     <string>&amp;About</string>
    </attribute>
    <widget class="QPushButton" name="exitBtn">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>370</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>&amp;Exit</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="helloBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>571</width>
       <height>471</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>ALOHA</string>
     </property>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>381</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Thank you for using this plain program!</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>140</y>
        <width>381</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Author: thrinkTu</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>260</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Current time:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>360</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Running time:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>310</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Boot time:</string>
      </property>
     </widget>
     <widget class="QLabel" name="cur_time">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>260</y>
        <width>361</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>current time</string>
      </property>
     </widget>
     <widget class="QLabel" name="boot_time">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>310</y>
        <width>361</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>boot time</string>
      </property>
     </widget>
     <widget class="QLabel" name="run_time">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>360</y>
        <width>361</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>running time</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="shutdownBtn">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>110</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Shut&amp;down</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rebootBtn">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>240</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Re&amp;boot</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_time">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>590</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Time</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_mem">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>590</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Mem</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_cpu">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>590</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CPU</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
